cnf-tests/hack/build-test-bin.sh

```diff
@@ -18,7 +18,7 @@ DONT_REBUILD_TEST_BINS="${DONT_REBUILD_TEST_BINS:-false}"
 
 if ! which ginkgo; then
 	echo "Downloading ginkgo tool"
-	go install -mod=readonly github.com/onsi/ginkgo/v2/ginkgo@v2.8.4
+	go install github.com/onsi/ginkgo/v2/ginkgo
```

Comment:
we pinned it to a reason though, because we were bitten in the past by compatibility changes. Building from vendor is good enough - we pin packages here using go.mod. But what guarantees we build from vendor?
perhaps we need `-mod=vendor` or something like it?

Reply:
This will not download latest, but the vendor version.
see this from https://go.dev/ref/mod#go-install:
`At go 1.14 or higher, automatic [vendoring](https://go.dev/ref/mod#vendoring) may be enabled. If the file vendor/modules.txt is present and consistent with go.mod, there is no need to explicitly use the -mod=vendor flag.`

Reply:
ok, good enough then.

Reply:
nit: the "downloading" comment is thus now misleading, because we are building from `vendor/`

Reply:
right, good point. let me reword

Reply:
I'd keep the mod=vendor too, to be on the safe side

Reply:
we don't need this on the `openshift-ci/Dockerfile.tools` I guess right?